### PR TITLE
DIV-5492 Add drop bulk case event

### PR DESCRIPTION
**Pull request by qzhou-hmcts**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: definitions/bulk-action/json/AuthorisationCaseState.json [hunk 1]
```diff
@@ -82,5 +82,12 @@
     "CaseStateID": "DNPronounced",
     "UserRole": "caseworker-divorce-superuser",
     "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "09/11/2018",
+    "CaseTypeID": "DIVORCE_BulkAction",
+    "CaseStateID": "dropped",
```
Comment (alexwakeman) [inline definitions/bulk-action/json/AuthorisationCaseState.json]: We usually us Captialized states, should we do that here too? i.e. `Dropped` just to help differentiate

File: definitions/bulk-action/json/AuthorisationCaseEvent.json [hunk 2]
```diff
@@ -110,5 +110,12 @@
     "CaseEventID": "cancelscheduleForListing",
     "UserRole": "caseworker-divorce-superuser",
     "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2019",
+    "CaseTypeID": "DIVORCE_BulkAction",
+    "CaseEventID": "drop",
```
Comment (alexwakeman) [inline definitions/bulk-action/json/AuthorisationCaseEvent.json]: Could be a little more descriptive maybe, like `dropBulkCase` or something?

File: definitions/bulk-action/json/AuthorisationCaseState.json (hunk 1 above)
Comment (alexwakeman) [reply definitions/bulk-action/json/AuthorisationCaseState.json]: Also could be `DroppedBulkCase` - as it is more informative from the user's perspective

Comment (qzhou-hmcts) [reply definitions/bulk-action/json/AuthorisationCaseState.json]: ah yup, Capitalised for consistency, about the `DroppedBulkCase` do yo mean more informative for devs? this is the Id the users won't see this, and they already have the bulk case context by given by the case type

Comment (alexwakeman) [reply definitions/bulk-action/json/AuthorisationCaseState.json]: Yeah for devs - I guess given the context it's ok

**Pull request merged**